src/libraries/System.Net.Requests/src/System/Net/HttpWebRequest.cs

```diff
@@ -1174,6 +1195,7 @@ private Task<HttpResponseMessage> SendRequest(bool async, HttpContent? content =
 
             _sendRequestMessage = new HttpRequestMessage(HttpMethod.Parse(_originVerb), _requestUri);
             _sendRequestCts = new CancellationTokenSource();
+            _sendRequestCts.CancelAfter(Timeout);
```

Comment:
```suggestion
```

The `Timeout` is already enforced by `HttpClient`

Reply:
Oh wait, these changes were not intentional, I'll remove them